python/paddle/v2/framework/tests/op_test_util.py

```diff
@@ -61,10 +61,7 @@ def test_all(self):
                 for out_name in func.all_output_args:
                     actual = numpy.array(scope.find_var(out_name).get_tensor())
                     expect = getattr(self, out_name)
-                    # TODO(qijun) The default decimal is 7, but numpy.dot and eigen.mul
-                    # has some diff, and could not pass unittest. So I set decimal 3 here.
-                    # And I will check this in future.
-                    numpy.testing.assert_almost_equal(actual, expect, decimal=3)
+                    numpy.isclose(actual, expect)
```

Comment:
An error of 0.00001 is appropriate for numbers around one, too big for numbers around 0.00001, and too small for numbers around 10,000. A more generic way of comparing two numbers – that works regardless of their range, is to check the relative error. Relative error is measured by comparing the error to the expected result.
Please refer to http://www.cygnus-software.com/papers/comparingfloats/Comparing%20floating%20point%20numbers.htm